### PR TITLE
chore: return to previous bundling strategy

### PR DESCRIPTION
**Pull request by MkMan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,16 +2,16 @@
...
   "description": "UI Component Library and Design System for Aēsop.com",
-  "main": "dist/index.js",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
   "types": "dist/index.d.ts",
   "files": [
     "dist"
   ],
   "scripts": {
-    "prepare": "npm run type-definitions && rollup -c",
```
Comment (MkMan) [inline package.json:11]: Do we need this given `prepublishOnly` runs the same command?

File: rollup-code-split.config.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    },
+    external: [...Object.keys(pkg.peerDependencies || {})],
+    plugins: [
+      // del({
+      //   targets: [
+      //     ...gelCategories.map((category) => `dist/${category}`),
+      //     'dist/sharedChunks',
+      //   ],
+      //   verbose: false,
+      // }),
```
Comment (MkMan) [inline rollup-code-split.config.js:68]: This is the only change I made in this file.

File: rollup.config.js [hunk 3]
```diff
@@ -1,119 +1,74 @@
...
-  alias({
-    entries: [{ find: '~', replacement: path.resolve(__dirname, 'src') }],
-  }),
-  postcss({
-    modules: {
-      sourceMap: true,
-      minimize: true,
-    },
-    // extract: 'styles.css',
-    plugins: [postcssDiscardComments()],
```
Comment (MkMan) [inline rollup.config.js:81]: Added this.

File: rollup.config.js [hunk 4]
```diff
@@ -1,119 +1,74 @@
...
     ],
-    external: Object.keys(pkg.peerDependencies),
-    plugins,
+    external: [...Object.keys(pkg.peerDependencies || {})],
+    plugins: [
+      // del({
+      //   // delete folders in dist root and assets folder only
+      //   targets: ['dist/*.*', 'dist/assets'],
+      //   verbose: false,
+      // }),
```
Comment (MkMan) [inline rollup.config.js:40]: Disabled this.

File: package.json (hunk 1 above)
Comment (maxlew) [reply package.json:11]: prepare should run during npm-link or if it's installed off a github branch

Comment (MkMan) [reply package.json:11]: I'll bring it back. Will this be ok?
```json
"prepare": "npm run build",
```

**Review by maxlew — APPROVED** [review]
Looks good, I'm assuming you've actually run it

File: package.json (hunk 1 above)
Comment (MkMan) [reply package.json:11]: Done

**Comment by MkMan** [comment]
> Looks good, I'm assuming you've actually run it

@hassanraza0054 tested it with the corresponding Web UI branch during the Web Chapter meeting and it worked 👍

**Pull request merged**